2017-04-11 01:29:10,888 - streamparse.stdout - INFO - hello world
2017-04-11 01:29:10,888 - streamparse.stdout - INFO - Database tcount exists already
2017-04-11 01:29:10,902 - streamparse.storm.component - ERROR - Exception in WordCounter.run()
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/streamparse-2.1.4-py2.7.egg/streamparse/storm/component.py", line 487, in run
    self.initialize(storm_conf, context)
  File "bolts/wordcount.py", line 57, in initialize
    count INT     NOT NULL);")
ProgrammingError: relation "tweetwordcount" already exists

